cartographer/mapping_2d/sparse_pose_graph/constraint_builder.cc

```diff
@@ -60,6 +60,16 @@ ConstraintBuilder::~ConstraintBuilder() {
   CHECK(when_done_ == nullptr);
 }
 
+/**
```

Comment:
This is not generallicaly a c++ convention (though it always makes sense IMHO). Cartographer follows the [Google c++ style guide](https://google.github.io/styleguide/cppguide.html) which is well written and contains good explanations for the conventions.